common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/BlockFetchingListener.java

```diff
@@ -17,11 +17,11 @@
 
 package org.apache.spark.network.shuffle;
 
-import java.util.EventListener;
-
 import org.apache.spark.network.buffer.ManagedBuffer;
 
-public interface BlockFetchingListener extends EventListener {
+public interface BlockFetchingListener extends BlockTransferListener {
+  String transferType = "fetch";
```

Comment:
I do not think the `transferType` (public static) members which only used  in `getTransferType()` methods are needed at all.

Reply:
Intention was to reuse the same string in a straightforward way when `RetryingBlockTransferor` repeatedly invokes `getTransferType()` on multiple push/fetch listeners during block transfer retry.

Reply:
The `transferType` now member val referencing the string literal "fetch" and when you use a string literal directly (i.e. "fetch") in the method (as that is the only place where it used) there should be no difference at all in performance. 

Like this

```
 @Override
  default String getTransferType() {
    return "fetch";
  }
```

It just works as fine but without an unnecessary member.